pkgs/top-level/all-packages.nix

```
@@ -688,6 +688,8 @@ in
     inherit (darwin.apple_sdk.frameworks) Carbon Cocoa Kernel;
   };
 
+  Archi = callPackage ../tools/misc/Archi { };
```

```suggestion
  archi = callPackage ../tools/misc/archi { };
```

Names must be lowercased at the beginning.